gtests/net/mptcp/common/multi-ep.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+#
+# Assign additional (advertised) addresses to the local device,
+# add a local/remote subflow for each address
+
+usage() { echo "$0 [-e <endpoints>] [-m <signal|subflow>] [-b]" 1>&2; exit 1; }
+
+create_endpoints() {
+    local host= flags= ep=1 max=$1 hn=$2 nn=$3 pl=$4 sig=${5:-0} bck=${6:-0}
+
+
+    [ $sig -eq 1 ] && flags=${flags:+$flags }signal || flags=${flags:+$flags }subflow
```

Comment:
I guess we don't need `${flags:+$flags }` here, only at the next line

Reply:
correct